src/components/PeoplePage.tsx

```diff
@@ -1,31 +1,138 @@
+import {
+  useEffect, useMemo, useState, FC,
+} from 'react';
+import { useSearchParams } from 'react-router-dom';
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { SortTypes } from '../types/SortTypes';
+import { QueryTypes } from '../types/QueryTypes';
+
+export const PeoplePage: FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasError, setHasError] = useState(false);
+
+  const [searchParams] = useSearchParams();
+  const query = searchParams.get(QueryTypes.Query) || '';
+  const sex = searchParams.get(QueryTypes.Sex) || '';
+  const centuries = searchParams.getAll(QueryTypes.Centuries) || [];
```

Comment:
If there is no such a params, centuries would be an empty array anyways

Don't worry. It's not worth requesting changes 😅 

```suggestion
  const centuries = searchParams.getAll(QueryTypes.Centuries);
```